### ISSUE TITLE
What do Top1 and Top5 mean respectively?

### ISSUE THREAD

**Issue by luoluo-gif** [open-mmlab/mmskeleton#278]
**What do Top1 and Top5 mean respectively? When I use my own dataset to train the model, Top1 and top5 are very different. Top 1 is only 40% and top 5 is 100%. Why?** @yysijie

**Comment by dusa2** [comment]
Top 1 means: whether the first predicted label is the correct one (usually what is important), and Top 5 means: whether one of the first, second, third, forth and fifth predictions (according to their confidence) are correct (used often as a supplementary information, especially meaningful if you have many classes) .

**Comment by luoluo-gif** [comment]
Hi,do you know why I get the result of top = 40%, top 5 = 100%? In my iteration, top 5 reached 100% early, but my top 1 has been hovering at a very low value?
I'm very confused about this situation, but I can't find the problem all the time?I’m eager to solve this,please give me some advice...
@yjxiong @dusa2 @lindahua @cclauss

**Comment by yjxiong** [comment]
I don't know how to help. Please provide more context when you ask questions. What dataset are you using? What is the number of classes? How many training/testing samples are there. Otherwise people will not be able to help you.

**Comment by luoluo-gif** [comment]
> I don't know how to help. Please provide more context when you ask questions. What dataset are you using? What is the number of classes? How many training/testing samples are there. Otherwise people will not be able to help you.

I used the Weizmann dataset with only ten categories, training set / test set = 8 / 1. @yjxiong

**Comment by yjxiong** [comment]
It is a pretty small dataset. You may not be able to train the model on it. Better extract features from the pretrained models and train an SVM for this task.
